### DISCUSSION TITLE
2019/03/14

### DISCUSSION THREAD

**Discussion by giscus** [doppelganger9/blog#274] [Post comments]
# 2019/03/14

A post about how I made emojis available on all OSes and browsers for reveal-md.

https://lacourt.dev/2019/03/14

**Comment by doppelganger9** [comment]
It is 2022 and I updated the sample to add GitHub Action with Github Pages hosting, PWA support, and the PDF export seems broken.

This blog from my 2019 self was totally useful to my 2022 self, so thank you, past-me!

**Reply by doppelganger9** [reply]
check it here : https://doppelganger9.github.io/sample-reveal-md-twemoji-presentation/
